docs/reference/mcp/overview.md

```
@@ -238,6 +238,31 @@ Success responses are normalized:
 }
 ```
 
+## Task-Augmented Calls
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)